includes/scheduler/class-comment.php

```diff
@@ -25,18 +25,7 @@ public static function init() {
 
 		// Comment transitions.
 		\add_action( 'transition_comment_status', array( self::class, 'schedule_comment_activity' ), 20, 3 );
```

Comment:
@pfefferle What's the reason for going with priority `20` rather than `10`?

Reply:
I wanted to get sure that all other hooks where called before we schedule the federation process.